bundles/org.openhab.binding.xmppclient/pom.xml

```
@@ -15,13 +15,14 @@
   <name>openHAB Add-ons :: Bundles :: XMPPClient Binding</name>
 
   <properties>
-    <smack.version>4.3.3</smack.version>
+    <smack.version>4.5.0-beta2</smack.version>
+    <bnd.importpackage>!sun.security.*</bnd.importpackage>
```

Did you try 

```suggestion
    <bnd.importpackage>sun.security.*;resolution:=optional</bnd.importpackage>
```

I have searched all poms for sun.security....

---

Thanks for joining on this issue, while this did notmake a difference, there is some progress.